### ISSUE TITLE
header and footer functionality of latest release (marp-cli)

### ISSUE THREAD

**Issue by tosaba** [yhatt/marp#243]
It seems that with the newest release of marp-cli, the header and footer functionality contains some incompatibilities. Regular text is shown as expected, but as soon as you use bold text or include images, the respective html code is shown, on both, html and PDF slides.

in markdown: 
``` markdown
---
header: "**bold** _test_"
footer: "![image](localimage.png)  footer title"
theme: base
---
```

on the slide:
``` html
<strong>bold</strong> <em>test</em>
...

<img src="localimage.png" alt="image" /> footer line
```

Thanks !

**Comment by yhatt** [comment]
Maybe it caused by the update of marp-core v0.0.5. (marp-team/marp-core#26)
Header and footer are [defined as HTML block](https://github.com/marp-team/marpit/blob/ea2a1c36ef305f4e5231ae279ab8781ca230a3a2/src/markdown/header_and_footer.js#L33) in the Marpit framework, so these elements might be sanitized unintentionally.

**Comment by yhatt** [comment]
Ready to use the updated `@marp-team/marp-cli` v0.0.7, it has included upgrades about the framework (Marpit v0.0.15) and core (marp-core 0.0.6).

**Comment by tosaba** [comment]
:+1:
